anta/tests/services.py

```diff
@@ -193,44 +204,35 @@ class VerifyErrdisableRecovery(AntaTest):
     class Input(AntaTest.Input):
         """Input model for the VerifyErrdisableRecovery test."""
 
-        reasons: list[ErrDisableReason]
+        reasons: list[ErrdisableRecovery]
         """List of errdisable reasons."""
-
-        class ErrDisableReason(BaseModel):
-            """Model for an errdisable reason."""
-
-            reason: ErrDisableReasons
-            """Type or name of the errdisable reason."""
-            interval: ErrDisableInterval
-            """Interval of the reason in seconds."""
+        ErrDisableReason: ClassVar[type[ErrdisableRecovery]] = ErrdisableRecovery
```

Comment:
```suggestion
        ErrDisableReason: ClassVar[type[ErrdisableRecovery]] = ErrDisableReason
```
Since we are chaning the name, we should keep `ErrDisableReason` in the input models with a deprecation, just like we do with other models (to be removed in ANTA v.2.0).

Reply:
Updated. Thanks!!